drivers/display/Kconfig.st7735r

```diff
@@ -0,0 +1,26 @@
+# ST7735R display driver configuration options
+
+# Copyright (c) 2020 Kim Bøndergaard <kim@fam-boendergaard.dk>
+# SPDX-License-Identifier: Apache-2.0
+
+menuconfig ST7735R
+	bool "ST7735R display driver"
+	depends on SPI
+	help
+	  Enable driver for ST7735R display driver.
+
+
+choice ST7735R_PIXEL_FORMAT
+	prompt "Color pixel format"
+	default ST7735R_RGB565
+	depends on ST7735R
+	help
+	  Specify the color pixel format for the ST7735R display controller.
+
+config ST7735R_BGR565
+	bool "BGR565"
+
+config ST7735R_RGB565
+	bool "RGB565"
```

Comment:
better move to DeviceTree with other display parameters

Reply:
Will do